doc/2/guides/getting-started/2-storing-data/index.md

```diff
@@ -4,4 +4,84 @@ type: page
 title: Storing Data
 description: Create and retrieve documents
 order: 200
----
\ No newline at end of file
+---
+
+# Storing Data
+
+Now that Kuzzle is running, we are going to create some documents.
+
+Kuzzle organizes the data storage in 4 levels: 
+  - indexes
+  - collections
+  - documents
+  - fields
+
+An index brings together several collections, which in turn contain several documents, each of which is composed of several fields.
+
+The collections have mappings that characterize the way Elasticsearch will index the data for the search.
+
+::: info
+It is recommended that you define your own mappings in order to specify the desired types for your collections and avoid default types that may be imprecise.
+:::
+
+If you're more familiar with the way relational databases store data, here is how it compares:
```

Comment:
I feel that we should be more cautious when comparing No-SQL databases with relational ones, as the two are really different beasts. Here is a proposal:

```suggestion
If you're more familiar with the way relational databases store data, here is an analogy. Bear in mind that this is only to give you a rough point of comparison with a relational database, the similarities end here:
```